simple/stats/reporter.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+from enum import auto
+from enum import Enum
+from functools import wraps
+import json
+import os
+import sys
+
+# For importing util
+_CODEDIR = os.path.dirname(os.path.realpath(__file__))
+sys.path.insert(1, os.path.join(_CODEDIR, "../"))
+
```

Comment:
I wonder should we start to add "__init__.py" and have module structure and not using sys path operation like this.

Then you can run main.py as module like "python3 -m simple.stats.main"

Reply:
That's a good idea. Let me do this in a separate PR.